### ISSUE TITLE
Kernel#sprintf('%.2f') is not accurate like Ruby 2.4+

### ISSUE THREAD

**Issue by deepj** [oracle/truffleruby#1511]
Since Ruby 2.4, `sprintf('%.2f')` returns a different accuracy.

MRI (2.5.3)

```ruby
sprintf('%.2f', '1.025')
# => "1.02"
```

TruffleRuby (1.0.0-rc10)

```ruby
sprintf('%.2f', '1.025')
# => "1.03"
```

**Comment by chrisseaton** [comment]
It's not inaccurate - it's rounding incorrectly.

**Comment by andrykonchin** [comment]
Fixed in c99ec964fb51f8364c6919467e898db6969f4058. Thank you for reporting.
